### PR TITLE
Gone pink with the TN

### PR DESCRIPTION
**Pull request by DonRichards**
**JIRA Ticket**: https://jira.duraspace.org/browse/ISLANDORA-2388

A known issue with ICC profile in Firefox (reported in 2010) causes some images to display as pink. 

> The QCMS color management system introduced in Firefox 3.5 only supports ICC version 2 color profiles, not version 4. [ICC color correction in Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/3.5/ICC_color_correction_in_Firefox)

Either ICC profiles and/or embedded issues with embedded tag values could cause these errors and can be easily stripped out of the TN. The ICC profile metadata embedded in the TN and any tags that aren't essential can be stripped out during generation/regeneration to correct this. 

# What does this Pull Request do?

1. Corrects the pink thumbnail issue for TN images displaying in Firefox.
2. Adds progressive scan to the TN image to improve page performance (slightly).

# What's new?
For islandora's large image create tn derivative
The "convert" command added these 2 arguments -strip -interlace Plane

# How should this be tested?
Here is an [example Tiff](https://jira.duraspace.org/secure/attachment/19254/icc_profile_causes_pink_tn.tiff) that causes this issue to test with.

1. Ingest large image objects (both jp2 & tif)
2. regenerate TN for both
Pull in this PR and go through the previous steps again in any order

# Additional Notes:

**Example of a working profile**
> Properties:
    icc:description: sRGB IEC61966-2.1
    icc:manufacturer: IEC http://www.iec.ch
    icc:model: IEC 61966-2.1 Default RGB colour space - sRGB

**Example of a profile that causes irregularity with some browsers**
> Properties:
    icc:description: SF_R (UMAX PowerLook 2100XL)

# Interested parties
@Islandora/7-x-1-x-committers

### CODE REVIEW

**Comment by DonRichards** [comment]
This is how an image can vary when a ICC profile is stripped. https://mdn.mozillademos.org/files/2669/iccsample.jpg

**Comment by DonRichards** [comment]
Adding a alpha channel to a TIF will cause the TN generation to fail. I created this failure prone file with Imagemagick. https://github.com/DonRichards/Islandora-Book-Batch-Example

**Comment by dannylamb** [comment]
Any reason @Islandora/7-x-1-x-committers can think of as to why these new arguments would cause any problems?  They seem like good improvements, but imagemagick goes deep and I don't pretend to know it all.

**Comment by DonRichards** [comment]
Just to recap what concerns were brought up in the original committer's call.

Concerned how this will impact thumbnails.
* The example image [here](https://mdn.mozillademos.org/files/2669/iccsample.jpg) shows different applications/browsers could interpreted an image that has the ICC stripped out. 
* This PR impacts new/regenerated TN images and should not impact existing TN files.

@ajstanley Would you have a second to comment on this?

**Comment by DonRichards** [comment]
@Islandora/7-x-1-x-committers Looking for some feedback and/or a review. We're wanting to roll this out onto our production site but would like feedback & adoption before doing so.

**Comment by DonRichards** [comment]
## TL;DR
This is about thumbnails keeping their ICC profile when generated. In most if not all Web browsers; an image's colors will be sent straight to the monitor unaltered, except Firefox with its "Full Color Management" which is assigning sRGB and converting it to monitor RGB with or without a ICC profile. When an image without an ICC profile being viewed on a web browser the RGB numbers are being sent straight to the monitor unaltered in most if not all Web browsers, except Firefox with its "Full Color Management" enabled which is converting it to Monitor RGB.
Most of the internet does not not embed their ICC profiles. The most common configuration is an untagged (without ICC profile) sRGB color space.

<hr>
I don't suggest having the TN generation to convert the colorspace, this might narrow the image's color range and the output can be unpredictable. Leaving the ICC profile stripped out allows the browser to pass the images straight to the monitor unaltered but converting a "whacked" profile will cause more unwanted issues.

Example

### What this means
* This solution should not have an impact on existing TNs (unless you regenerate TNs).
* This should have nearly no visible impact on your TNs unless you're currently seeing pink TNs already in Firefox, and if so, it should correct that after regenerating.

**Comment by DonRichards** [comment]
FYI: IIIF strips this by default.

**Review by bondjimbond — APPROVED** [review]
I don't understand it, but I have tested and it works. 24 hour clock ticking now.

**Comment by DonRichards** [comment]
@bondjimbond Ready to merge this?
